src/async-components/views/dialogs/keybackup/CreateKeyBackupDialog.js

```diff
@@ -484,19 +471,19 @@ export default React.createClass({
     _titleForPhase: function(phase) {
         switch (phase) {
             case PHASE_PASSPHRASE:
-                return _t('Create a Recovery Passphrase');
+                return _t('Secure your backup with a passphrase');
             case PHASE_PASSPHRASE_CONFIRM:
-                return _t('Confirm Recovery Passphrase');
+                return _t('Confirm your passphrase');
             case PHASE_OPTOUT_CONFIRM:
                 return _t('Warning!');
             case PHASE_SHOWKEY:
-                return _t('Recovery Key');
+                return _t('Recovery key');
```

Comment:
We capitalize `Key` elsewhere on buttons and  things - why lowercase it here?

Reply:
We've been treating Key Backup as a proper noun, and recovery key as a common noun.

Might be worth reflecting on when we've got a bit more breathing space, but I'm happy with that decision for now :)

Reply:
Where 'happy with that decision' == leave it how it is I mean.